NOTES.rst

```diff
@@ -39,6 +39,35 @@ The version requirements have changed for these dependencies:
 * :mod:`python-ldap` must be any version of the *3.0* series.
 
 
+Backwards incompatible changes
+------------------------------
+
+Changed command line options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Some of the NAV programs have changed their command line interface:
+
+* :program:`alertengine.py`: The nonworking ``--loglevel`` option was removed.
+* :program:`pping.py`: The ``-n/--nofork`` option was renamed to ``-f/--foreground``.
+* :program:`servicemon.py`: The ``-n/--nofork`` option was renamed to ``-f/--foreground``.
+* :program:`smsd.py`: The ``-n/--nofork`` option was renamed to
+  ``-f/--foreground``, while the ``-f/--factor`` option was renamed to
+  ``-D/--delayfactor``.
```

Comment:
Indeed better names. How do you plan to flag this in the documentation? Especially that `-f` means something new for `smsd`.

Reply:
This _is_ part of the documentation, @hmpf :-)